CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Change Log
```

Comment:
Nice to see this standardized way of communicating the changelog arrive in more repositories 👍